authorizer-app-backend/src/main/resources/db/changelog/changes/db.changelog-master.xml

```diff
@@ -7,4 +7,5 @@
   <include  file="db/changelog/changes/00000000000000_initial_schema.xml"/>
   <include  file="db/changelog/changes/00000000000001_update_schema.xml"/>
   <include  file="db/changelog/changes/00000000000002_update_schema.xml"/>
+  <include  file="db/changelog/changes/00000000000003_update_schema.xml"/>
```

Comment:
I see the same thing has happened in previous files, but could you please rename the file to `00000000000003_add_created_at_column.xml`? That way if we ever need to troubleshoot the table layout it's easier to locate the issue.